README.rst

```diff
@@ -154,10 +154,12 @@ Then we convert ``MyGreatProject.jucer`` to a new ``CMakeLists.txt`` file: ::
 
   $ cd <root>/MyGreatProject/
 
-  $ ../FRUT/prefix/FRUT/bin/Jucer2Reprojucer MyGreatProject.jucer ../FRUT/prefix/FRUT/cmake/Reprojucer.cmake
+  $ ../FRUT/prefix/FRUT/bin/Jucer2Reprojucer MyGreatProject.jucer ../FRUT/prefix/FRUT/cmake/Reprojucer.cmake --juce-modules="../JUCE/modules"
 
   <root>/MyGreatProject/CMakeLists.txt has been successfully generated.
 
+*Note that the* ``--juce-modules`` *argument is only necessary if* ``MyGreatProject.jucer`` *uses modules from the global "JUCE Modules" path set in Projucer.*
```

Comment:
Please break this line into several ones, so they don't go over 90 characters.
I know that many lines in this file don't respect that rule, but it's either to accommodate the reStructuredText syntax or to make inline HTML easier to read.